### PR TITLE
6196 tweak dashboard headers

### PR DESCRIPTION
**Pull request by emitche**
- Adjust heading margins and padding in sections in dashboard

### CODE REVIEW

**Comment by rgalusca** [comment]
@emitche, maybe just a little bit more space between the headers and the tables in the case of My Projects, Upcoming Steps, and Logs. I think now they are too close. Users look good. Thank you!

**Comment by emitche** [comment]
@rgalusca See the updated styling. Thank you!

image

**Comment by rgalusca** [comment]
Looks great! Thanks @emitche!

File: app/views/admin/dashboard/dashboard.html.slim [hunk 1]
```diff
@@ -40,10 +40,11 @@
...
       - @people.each do |person|
-        h3 = person.name
-        .recent-projects
-          = render "recent_projects_grid_definition", grid: @people_grids["#{person}"]
-          = render_index_grid(@people_grids["#{person}"])
+        .person
+          h3 = person.name
+          .recent-projects
+            = render "recent_projects_grid_definition", grid: @people_grids["#{person}"]
+            = render_index_grid(@people_grids["#{person}"])
```
Comment (hooverlunch) [inline app/views/admin/dashboard/dashboard.html.slim]: I must have missed this before but `"#{person}"` is kind of an anti-pattern. Just `person` or maybe `person.to_s` is more appropriate. If this isn't clear let me know.

**Pull request merged**